bundles/org.openhab.binding.bluetooth.bluegiga/src/main/resources/ESH-INF/thing/bluegiga.xml

```diff
@@ -20,6 +20,60 @@
 				<advanced>true</advanced>
 				<default>true</default>
 			</parameter>
+			<parameter name="passiveScanIdleTime" type="integer" min="100" max="60000">
+				<label>Passive Scan Idle Time</label>
+				<description>Passive scan idle time defines the time how long to wait in milliseconds before start passive scan.</description>
+				<advanced>true</advanced>
+				<default>5000</default>
+			</parameter>
+			<parameter name="passiveScanInterval" type="integer" min="4" max="16384">
+				<label>Passive Scan Interval</label>
+				<description>Passive scan interval defines the interval when scanning is re-started in units of 625us. Default is 4000ms.</description>
+				<advanced>true</advanced>
+				<default>6400</default>
+			</parameter>
+			<parameter name="passiveScanWindow" type="integer" min="4" max="16384">
+				<label>Passive Scan Window</label>
+				<description>Passive scan Window defines how long time the scanner will listen on a certain frequency and try to pick up advertisement packets. Default is 4000ms.</description>
+				<advanced>true</advanced>
+				<default>6400</default>
+			</parameter>
+			<parameter name="activeScanInterval" type="integer" min="4" max="16384">
+				<label>Active Scan Interval</label>
+				<description>Active scan interval defines the interval when scanning is re-started in units of 625us. Default is 4000ms.</description>
+				<advanced>true</advanced>
+				<default>6400</default>
+			</parameter>
+			<parameter name="activeScanWindow" type="integer" min="4" max="16384">
+				<label>Active Scan Window</label>
+				<description>Active scan Window defines how long time the scanner will listen on a certain frequency and try to pick up advertisement packets. Default is 4000ms.</description>
+				<advanced>true</advanced>
+				<default>6400</default>
+			</parameter>
+			<parameter name="connIntervalMin" type="integer" min="6" max="3200">
+				<label>Minimum Connection Interval</label>
+				<description>Minimum connection interval in units of 1.25ms. Default is 100ms.</description>
+				<advanced>true</advanced>
+				<default>80</default>
+			</parameter>
+			<parameter name="connIntervalMax" type="integer" min="6" max="3200">
+				<label>Maximum Connection Interval</label>
+				<description>Maximum connection interval in units of 1.25ms. Default is 1000ms.</description>
```

Comment:
why intervals of 1.25ms?

Reply:
It's according to BlueGiga specification